tests/ltp/buildenv.sh

```diff
@@ -45,6 +46,11 @@ if [[ "$mode" == "build" ]]; then
         git apply $FORK_DISABLE_PATCH
     fi
 
+    if [ -f $FIX_FLOCK_PATCH ];then
```

Comment:
Is this check necessary?

Reply:
Removed the check and updated the buildenv.sh file